src/components/HomePage.js

```diff
@@ -46,19 +46,17 @@ const HomePage = () => {
             No coding experience required
           </p>
 
-          <div
-            className="flex justify-center items-center  max-w-md rounded-full px-7 my-6 shadow-lg 
-          bg-gradient-to-tr from-orange-400 to bg-yellow-200 hover:scale-105"
-          >
-            <a
+          <a
               href="https://www.meetup.com/code-for-hawaii/"
```

Comment:
Not related to your changes, but 🚀 

```suggestion
              href="https://www.meetup.com/code-with-aloha/"
```

Reply:
love following the scout rule, leave it better than you found it, but is there a standard for making it a separate commit? for example, in case there's reason to revert the change described in the PR, don't want this one glued to it

Reply:
No set standard, we aren't that strict :P. But yes, usually if the change can break something and is being deployed somewhere, convention dictates breaking up your changes into easily revertible commits. Simple text changes like the above are usually safe, but any routing changes should probably be done in another PR 🐰